src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -445,6 +445,7 @@ public class BuildTransactionRequest : TxFeeEstimateRequest, IValidatableObject
         /// <summary>
         /// Whether to send the change to a P2WPKH (segwit bech32) addresses, or a regular P2PKH address
         /// </summary>
+        [DefaultValue(false)]
```

Comment:
This is an unrelated but necessary change that will be cherry-picked to the latest feature branch too